internal/config.go

```diff
@@ -5,11 +5,13 @@ import (
 	"path/filepath"
 )
 
+var HomeDir = getUserHomeDir()
+
```

Comment:
I started this work before https://github.com/CryoByte33/steam-deck-utilities/pull/65 but we ended up with basically the same implementation. I need this to work within the flatpak